src/components/pages/__test__/SearchPatient.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import renderer from 'react-test-renderer'
+import configureMockStore from 'redux-mock-store'
+import { Provider } from 'react-redux'
+import SearchPatient from '../SearchPatient';
+
+let props, mountedComponent, store;
+
+const mockStore = configureMockStore();
+const searchPatient = () => {
+    if (!mountedComponent) {
+        mountedComponent = shallow(
+            <Provider store={store}>
```

Comment:
Is there a reason you did a "shallow" here instead of a mount?

Reply:
When using mount it try to render the component with child and i get an error.